### ISSUE TITLE
Warn on unused hints

### ISSUE THREAD

**Issue by LeventErkok** [ndmitchell/hlint#1651]
Some of my files have a bunch of hlint annotations at the bottom like so:

```haskell
{- HLint ignore type FPOp "Use camelCase" -}
{- HLint ignore type PBOp "Use camelCase" -}
{- HLint ignore type OvOp "Use camelCase" -}
{- HLint ignore type NROp "Use camelCase" -}
```

These are collected over the years as I respectfully disagreed with hlint. But, as things evolve, some of these annotations become irrelevant in time: For instance I removed the `NROp` from that file a while ago but failed to remove the corresponding hint.

This is not a huge deal breaker, but I'm wondering if there's a switch to hlint which can "warn" me of unused module/function hints like this. Just like it warns about unused GHC language pragmas.

If such a feature doesn't exist, would it be costly to add it? Just a nicety, no big deal if it's a lot of work.
